dds/DdsDcpsCore.idl

```diff
@@ -300,6 +300,25 @@ module DDS {
 
     // ----------------------------------------------------------------------
 
+	  typedef unsigned long ParticipantLocation;
+
+  	const ParticipantLocation LOCATION_LOCAL = 0x0001 << 0;
+	  const ParticipantLocation LOCATION_ICE = 0x0001 << 1;
+	  const ParticipantLocation LOCATION_RELAY = 0x0001 << 2;
+
+  	BUILT_IN_TOPIC_TYPE
+	  struct ParticipantLocationBuiltinTopicData {
+		  BUILT_IN_TOPIC_KEY string guid;
```

Comment:
String GUIDs aren't used anywhere else in the API, can we make a case for using them here?